main.go

```diff
@@ -119,6 +121,9 @@ func main() {
 	stringFlagOrEnv(&neuronMonitorImage, "neuron-monitor-image", "RELATED_IMAGE_NEURON_MONITOR", fmt.Sprintf("%s:%s", neuronMonitorImageRepository, v.NeuronMonitor), "The default Neuron monitor image. This image is used when no image is specified in the CustomResource.")
 	pflag.Parse()
 
+	// set instrumentation config in environment variable to be used for default instrumentation
+	os.Setenv("AUTO_INSTRUMENTATION_CONFIG", autoInstrumentationConfigStr)
```

Comment:
I'm not opposed to using an env variable since we already are using it for the sdks....seems like it's not trivial to pass these args to the pod mutator.

However I'd prefer if we do the json parsing here and then set the env variables to the actual cpu and memory limit itself:

```
AUTO_INSTRUMENTATION_LIMIT_CPU=100m
AUTO_INSTRUMENTATION_LIMIT_MEMORY=64Mi
```

Having JSON in the env variable itself might have some weird validation or injection issues.